src/RA_Defs.cpp

```diff
@@ -76,6 +77,13 @@ std::wstring Widen(const wchar_t* wstr)
 
 _Use_decl_annotations_ std::wstring Widen(const std::wstring& wstr) { return wstr; }
 
+_Use_decl_annotations_
+std::wstring Widen(std::wstring&& wstr) noexcept
+{
+    const auto _wstr{ std::move_if_noexcept(wstr) };
+    return Widen(_wstr);
+}
```

Comment:
When demacrofying `NativeStr`, stuff was going to the wrong function. Made more no-ops to prevent reference dangling.